access-control-app/int-test/cmr/access_control/int_test/permission_check_test.clj

```diff
@@ -920,6 +921,59 @@
             "user1" ["update" "delete"]
             "user2" []))))))
 
+(defn get-permissions-2
```

Comment:
nit, get-permissions-2 needs to be descriptive.  Maybe get-user-concept-id-permissions or something like that.  Mostly just remove the -2